examples/run_provider_tests.sh

```
@@ -0,0 +1,117 @@
+             key=${MAESTRO_TEST_SIGNING_KEY:-$TEST_SIGNING_KEY}
+        elif [ "$idx" -eq 3 ]; then
+             addr=${KOIOS_TEST_ADDRESS:-$TEST_ADDRESS}
+             key=${KOIOS_TEST_SIGNING_KEY:-$TEST_SIGNING_KEY}
+        fi
+    fi
+    
+    echo "$addr $key"
+}
+
+# Fetch Credentials for Mint (Wallet 1) and Spend (Wallet 2)
+read mint_addr mint_key <<< $(get_wallet_credentials 1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)